src/plugins/discover/public/application/apps/main/components/sidebar/discover_field.tsx

```diff
@@ -99,205 +265,43 @@ export function DiscoverField({
   onEditField,
   onDeleteField,
 }: DiscoverFieldProps) {
-  const addLabelAria = i18n.translate('discover.fieldChooser.discoverField.addButtonAriaLabel', {
-    defaultMessage: 'Add {field} to table',
-    values: { field: field.name },
-  });
-  const removeLabelAria = i18n.translate(
-    'discover.fieldChooser.discoverField.removeButtonAriaLabel',
-    {
-      defaultMessage: 'Remove {field} from table',
-      values: { field: field.name },
-    }
-  );
-
   const [infoIsOpen, setOpen] = useState(false);
 
-  const toggleDisplay = (f: IndexPatternField, isSelected: boolean) => {
-    if (isSelected) {
-      onRemoveField(f.name);
-    } else {
-      onAddField(f.name);
-    }
-  };
+  const toggleDisplay = useCallback(
+    (f: IndexPatternField) => {
```

Comment:
:information_source: I changed that signature to use `selected` instead of explicitly pass in `isSelected` when calling it. Functionality should be the same.